javascript/javascript1/week2/formalname.js

```diff
@@ -0,0 +1,47 @@
+let fullName;
+let firstName;
+let surName;
+let useFormalName;
+
+function getFullname(firstName, surName, useFormalName) {
+    if ((firstName == null) || (firstName == '') || (firstName.indexOf(' ') >= 0) ){
```

Comment:
You don't need the extra parentheses here:
```js
    if (firstName == null || firstName == '' || firstName.indexOf(' ') >= 0){
```

Reply:
You can check for missing value like this:
`if(!firstName)` that way, you can avoid the two first cases.